impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -49,138 +63,134 @@ public class KafkaClusterTest {
     private static final Duration CLUSTER_FORMATION_TIMEOUT = Duration.ofSeconds(10);
     private TestInfo testInfo;
     private KeytoolCertificateGenerator keytoolCertificateGenerator;
+    private static int clusterSizesIndex = 0;
 
-    @Test
-    public void kafkaClusterKraftMode() throws Exception {
-        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder()
-                .testInfo(testInfo)
-                .kraftMode(true)
-                .build())) {
-            cluster.start();
-            verifyRecordRoundTrip(1, cluster);
-        }
+    private static Stream<Version> versions() {
+        return Stream.of(
+                version("latest")
+        // TODO: waiting for new versions support in ozangunalp repo https://github.com/ozangunalp/kafka-native/issues/21
+        // version("3.3.1"),
```

Comment:
Is there a version other than `latest` we could use now before https://github.com/ozangunalp/kafka-native/issues/21 is fixed?

Reply:
Not in the format we talked about. There is no image in quay pointing to a concrete kafka version as we expected, so we cannot currently point to any of the images following that approach